aws_fuzzy_finder/main.py

```diff
@@ -85,38 +89,47 @@ def entrypoint(use_private_ip, key_path, user, ip_only, no_cache, tunnel, tunnel
         LIBRARY_PATH
     )
 
-    username = ENV_SSH_USER or user or ''
-    if username:
-        username = '%s@' % (username)
+    chosen_host = choice(fuzzysearch_bash_command, use_ssm)
 
-    key = ENV_KEY_PATH or key_path or ''
-    if key:
-        key = '-i %s' % (key)
-
-    chosen_host = choice(fuzzysearch_bash_command)
-
-    if ip_only:
-        sys.stdout.write(chosen_host)
-        exit(0)
-
-    ssh_command = ENV_SSH_COMMAND_TEMPLATE.format(
-        user=username,
-        key=key,
-        host=chosen_host,
-    )
-
-    if tunnel:
-        ssh_command += " -t " + ENV_SSH_COMMAND_TEMPLATE.format(
-            user=ENV_TUNNEL_SSH_USER or tunnel_user,
-            key=ENV_TUNNEL_KEY_PATH or tunnel_key_path,
-            host=choice(fuzzysearch_bash_command),
+    if use_ssm or ENV_USE_SSM:
```

Comment:
Added a conditional branch to just incorporate all the ssh vs ssm logic in one area